data/data_80E2068.s

```diff
@@ -2281,8 +2281,8 @@ gUnknown_80E7684: @ 80E7684
         .string "Resuming saved adventure.\0"
         .align 2,0
 
-	.global gUnknown_80E76B8
-gUnknown_80E76B8: @ 80E76B8
+	.global gQuitWaitingResuceMenuItems
```

Comment:
Is this supposed to be "Rescue"?

Reply:
Yup, nice catch. Did a grep and fixed that with a follow-up commit.